server/services/event-handlers.js

```diff
@@ -15,11 +16,16 @@ const createCaptureFeatureHandler = (async (message) => {
 });
 
 const createRawCaptureFeatureHandler = (async (message) => {
-  const newRawCaptureFeature = rawCaptureFeatureFromMessage({ ...message });
-  const dbSession = new Session();
-  const rawCaptureFeatureRepo = new RawCaptureFeatureRepository(dbSession);
-  const executeRawCreateCaptureFeature = createRawCaptureFeature(rawCaptureFeatureRepo);
-  executeRawCreateCaptureFeature(newRawCaptureFeature);
+  try{
+    log.warn("createRawCaptureFeatureHandler...", message);
+    const newRawCaptureFeature = rawCaptureFeatureFromMessage({ ...message });
+    const dbSession = new Session();
+    const rawCaptureFeatureRepo = new RawCaptureFeatureRepository(dbSession);
+    const executeRawCreateCaptureFeature = createRawCaptureFeature(rawCaptureFeatureRepo);
+    await executeRawCreateCaptureFeature(newRawCaptureFeature);
+  }catch(e){
+    log.error("Get error when handling message:", e);
+  }

```

Comment:
Must wrap the code with try/catch, or we will miss the error message when things go wrong.